packages/http/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @turnkey/http
 
+## 1.1.2
+
+### Patch Changes
+
+- Sync protos
```

Comment:
I think this should be a minor release because we're introducing a new capability: create sub-org with keys. Also think we need a more detailed changelog entry to let people know that they'll need to update their code to look for createSubOrganization results in `createSubOrganizationV3` going forward.